project/AkkaDisciplinePlugin.scala

```diff
@@ -30,6 +30,7 @@ object AkkaDisciplinePlugin extends AutoPlugin {
     "akka-cluster",
     "akka-cluster-metrics",
     "akka-cluster-sharding",
+    "akka-cluster-sharding-typed",
```

Comment:
what does looseProjects mean?

Reply:
Commit adding this list talks about allowing return values to be discarded. @raboof ?

Reply:
can we disable that warning altogether? seems we are listing pretty much all projects here?